XmlFormatterOsIndependent/ViewModels/MainWindowViewModel.cs

```diff
@@ -0,0 +1,399 @@
+                        );
+                    content = stringBuilder.ToString();
+                    buttons = ButtonEnum.YesNo;
+                }
+                parameter.ContentTitle = title;
+                parameter.ContentMessage = content;
+                parameter.ButtonDefinitions = buttons;
+
+                IMsBoxWindow<ButtonResult> window = MessageBoxManager.GetMessageBoxStandardWindow(parameter);
+                TaskAwaiter<ButtonResult> awaiter = window.ShowDialog(view.GetWindow()).GetAwaiter();
+                awaiter.OnCompleted(() =>
+                {
+                    ButtonResult buttonResult = awaiter.GetResult();
+                    if (buttonResult == ButtonResult.Yes)
+                    {
+                        ICommand command = new UpdateApplicationCommand();
+                        ExecuteCommand(command, new UpdateApplicationData(pluginManager, settingsManager, versionInfo));
+                    }
+                });
+            }
+        }
+
+        /// <summary>
+        /// Open the about window
+        /// </summary>
+        public void OpenAboutCommand()
+        {
+            IDataCommand command = new OpenAboutCommand();
+            ExecuteAsyncCommand(command, view);
+        }
+
+        /// <summary>
+        /// Open the settings window
+        /// </summary>
+        public void OpenSettingsCommand()
+        {
+            IDataCommand command = new OpenSettingsCommand();
+            command.Executed += ShowSettings_Executed;
+            ExecuteAsyncCommand(command, view);
+        }
+
+        /// <summary>
+        /// Event if the settings window was shown
+        /// </summary>
+        /// <param name="sender">The sender of the event</param>
+        /// <param name="e">The parameters of this event</param>
+        private void ShowSettings_Executed(object sender, EventArgs e)
+        {
+            ChangeTheme();
+        }
+
+        /// <summary>
+        /// Save was executed successfully
+        /// </summary>
+        /// <param name="sender">The event sender</param>
+        /// <param name="e">The event arguments</param>
+        private void SaveCommand_Executed(object sender, System.EventArgs e)
+        {
+            if (sender is IDataCommand command)
+            {
+                if (!command.IsExecuted())
+                {
+                    return;
+                }
+
+                IFormatter formatter = pluginManager.LoadPlugin<IFormatter>(currentPlugin);
+
+                formatter.StatusChanged += Formatter_StatusChanged;
+                formatter.ConvertToFormat(CurrentFile, command.GetData<string>(), GetConvertionMode());
+            }
+        }
+
+        /// <summary>
+        /// Get the current conversion mode
+        /// </summary>
+        /// <returns>The mode of conversion</returns>
+        private ModesEnum GetConvertionMode()
+        {
+            return currentMode == 0 ? ModesEnum.Formatted : ModesEnum.Flat; ;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Reformat the code to have only one statement per line.](https://app.codacy.com/manual/simonaberle/XmlFormatter/pullRequest?prid=5634024)